Lib/random.py

```diff
@@ -239,7 +239,13 @@ def _randbelow_with_getrandbits(self, n):
         "Return a random int in the range [0,n).  Defined for n > 0."
 
         getrandbits = self.getrandbits
-        k = n.bit_length()  # don't use (n-1) here because n can be 1
+        # For efficiency, we'd like to use k = (n-1).bit_length() here, but
```

Comment:
Let's just remove the comment entirely.

Reply:
That sounds like an improvement over the status quo of having a misleading comment to me.

I guess we can stick the more in-depth explanation in the commit description, so at least it shows up in git blame?